app/src/main/java/io/github/dkter/aaaaa/AaaaaInputMethodService.kt

```diff
@@ -10,15 +10,28 @@ package io.github.dkter.aaaaa
 
 import android.inputmethodservice.InputMethodService
 import android.text.TextUtils
-import android.util.Log
 import android.view.KeyEvent
 import android.view.View
 import android.view.inputmethod.EditorInfo
 import android.view.inputmethod.InputConnection
+import kotlin.concurrent.thread
+
+class AaaaaInputMethodService : InputMethodService(), AaaaaKeyboardView.AaaaaKeyboardListener {
+
+    private fun newLongPressThread() = thread {
+        while (!Thread.currentThread().isInterrupted) {
+            val uppercase = (0..1).random() == 1
+            inputChar(if (uppercase) 'A' else 'a')
```

Comment:
I definitely want this to be an option. It wouldn't take that long for me to implement myself though - if you want to do it yourself, [here's the commit](https://github.com/dkter/aaaaa/commit/9b670ca3490ae3dad4147597d782d21b76cb3390) where I added the last settings option for reference.

Reply:
Instead of a setting, what I would prefer is an uppercase button (as suggested in #7) which determines the long press output as well.

With this, the user can type "AAAA", "aaaa" or "AAAaaAAaaaA" without having to open the settings.

I can try to add a caps lock button in a seperate pull request, if you agree with me. If you don't, I can implement it your way as well.

Reply:
Yeah, I think a caps lock button would make sense, if you're willing to implement that.

Reply:
I've set it to only lowercase 'a's for now (9c916b4). In the future, a caps lock button can determine whether it's 'a' or 'A'.